pulsar-client-cpp/lib/Message.cc

```diff
@@ -223,5 +223,8 @@ PULSAR_PUBLIC std::ostream& operator<<(std::ostream& s, const Message& msg) {
       << ", msg_id=" << msg.getMessageId() << ", props=" << msg.getProperties() << ')';
     return s;
 }
+KeyValue Message::getKeyValueData(const KeyValueEncodingType& keyValueEncodingType) const {
+    return KeyValue(getDataAsString(), keyValueEncodingType);
```

Comment:
Here it seems to result in a data copy. We may need to use `getData` here.

Reply:
Fixed.